java_gen/pre-written/src/test/java/org/projectfloodlight/openflow/types/IPAddressTest.java

```diff
@@ -92,6 +92,16 @@ public void testFromInetAddressException() throws UnknownHostException {
     }
 
     @Test
+    public void testToInetAddress() throws Exception {
+        IPAddress<?> ip0 = IPAddress.of("201.202.3.4");
+        IPAddress<?> ip1 = IPAddress.of("2001:db8:abcd::1:2:3:4");
+        InetAddress ia0 = ip0.toInetAddress();
+        InetAddress ia1 = ip1.toInetAddress();
+        assertEquals(ia0, InetAddress.getByName("201.202.3.4"));
```

Comment:
One thing that might be good to double check is that we don't trigger any network interaction here (i.e., this unit test doesn't hang forever if you take the computer off the Internet).

I am always a little bit scared of those InetAddress classes talking to the net by themselves...?